crates/core_arch/src/arm_shared/neon/mod.rs

```diff
@@ -4811,7 +6687,7 @@ pub unsafe fn vpadalq_u32(a: uint64x2_t, b: uint32x4_t) -> uint64x2_t {
 #[target_feature(enable = "neon")]
 #[cfg_attr(target_arch = "arm", target_feature(enable = "v8"))]
 #[cfg_attr(all(test, target_arch = "arm"), assert_instr(nop))]
-#[cfg_attr(all(test, target_arch = "aarch64"), assert_instr(smmla))]
+#[cfg_attr(all(test, target_arch = "aarch64"), assert_instr(nop))]
```

Comment:
Why is this changed to nop? This is incorrect.

Also why is this function not marked as stable?

Reply:
Same as above. And vmmla instructions require `v8.6a` architecture, so they are not included in this stabilization.